src/Qir/Tools/Microsoft.Quantum.Qir.Runtime.Tools.csproj

```diff
@@ -74,32 +65,4 @@
     </Compile>
   </ItemGroup>
 
-  <ItemGroup>
```

Comment:
Is this logic no longer needed? It seemed pretty important before to make sure the QSimDll got into the right folder. And to get the native resources as well.

Reply:
You're right. The e2e tests fail because of this. Even though this logic is important, it shouldn't be part of this project file. The problem with this is that it only works when the project is referenced directly and not when the package produced by this project is being referenced.

I'll fix this problem by adding a reference to the `Microsoft.Quantum.Simulators` package to the command line tool project and change some logic there if needed.

Reply:
Done.